docs/pages/inboxes/group-permissions.mdx

```diff
@@ -57,7 +58,16 @@ As the app developer, you can provide a UI that enables group participants to ma
   <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/group-perm-toggles.png" width="300px" />
 </div>
 
-You can use member statuses, options, and permissions to create a custom policy set.
+You can use member statuses, options, and permissions to create a custom policy set. The following table represents the valid policy options for each of the permissions:
```

Comment:
Code / source of truth for these can be found here => https://github.com/xmtp/libxmtp/blob/7d863bde76a751955b6719aee6cbaa86716a82d7/xmtp_mls/src/groups/intents.rs#L495-L535

Reply:
This TABLE! So good, so clear... thank you, @cameronvoell!